packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/text/ReactTextShadowNode.java

```diff
@@ -198,6 +198,11 @@ private Layout measureSpannedText(Spannable text, float width, YogaMeasureMode w
     Layout layout;
     BoringLayout.Metrics boring = BoringLayout.isBoring(text, textPaint);
     float desiredWidth = boring == null ? Layout.getDesiredWidth(text, textPaint) : Float.NaN;
+    // StaticLayout#getLineWidth does not work with single-line text.
+    boolean overrideTextBreakStrategySingleLine =
+        boring == null
+            ? false
+            : mNumberOfLines == 1 && !mAdjustsFontSizeToFit && boring.width > width;
```

Comment:
What about
```suggestion
    // StaticLayout#getLineWidth does not work with single-line text.
    boolean overrideTextBreakStrategySingleLine =
        boring != null && mNumberOfLines == 1 && !mAdjustsFontSizeToFit && boring.width > width;
```
and we avoid the ternary?